### ISSUE TITLE
[Feature] Add repo/website url column to the upgrade-interactive plugin

### ISSUE THREAD

**Issue by tim-soft** [yarnpkg/berry#3008]
- [X] I'd be willing to implement this feature ([contributing guide](https://yarnpkg.com/advanced/contributing))
- [X] This feature is important to have in this repository; a contrib plugin wouldn't do

**Describe the user story**

As a developer who has used several interactive upgrade tools (like [npm-check](https://github.com/dylang/npm-check) and yarn v1 upgrade-interactive), a big part of the upgrade process is doing due diligence --  first looking at each package's respective repos/websites, finding a changelog, and figuring out what the breaking changes/new features are.

The two tools I listed above provide urls in the terminal that I can `ctrl + click` to easily look up this information, and I sorely miss this feature in the yarn berry `upgrade-interactive` plugin

**Describe the solution you'd like**

Show the package repo/homepage url as a new column in `upgrade-interactive`

**Describe the drawbacks of your solution**

none

**Comment by latipun7** [comment]
Yeah I agree. It would be really useful to show the link to the package's repo or changelog. While yarn v1 upgrade-interactive showing the package's homepage URL.

Since yarn berry has something cool with the hyperlinks to show the error codes:
![error codes](https://i.imgur.com/AiRF55c.png)
it would be cool too to place the package's repo or changelog in the package's name 👍 
![hyperlink](https://i.imgur.com/JvDVgQQ.png)

**Comment by CanRau** [comment]
Yes please, very much needed 😍
